content/post/kubedb-v2024.4.27/index.md

```diff
@@ -0,0 +1,431 @@
+---
+title: Announcing KubeDB v2024.4.27
+date: "2024-04-30"
+weight: 14
+authors:
+- Raihan Khan
+tags:
+- alert
+- backup
+- cloud-native
+- dashboard
+- database
+- druid
+- grafana
+- kafka
+- kafka-connect
+- kubedb
+- kubernetes
+- kubestash
+- memcached
+- mongodb
+- mssql
+- pgpool
+- postgres
+- postgresql
+- prometheus
+- rabbitmq
+- redis
+- restore
+- s3
+- security
+- singlestore
+- solr
+- tls
+- zookeeper
+---
+
+We are pleased to announce the release of [KubeDB v2024.4.27](https://kubedb.com/docs/v2024.4.27/setup/). This release includes features like TLS support for SingleStore & PgPool; Monitoring & Alerting Support for Druid & Solr using Prometheus and Grafana; PDB support for Kafka, Kafka Connect Cluster and RabbitMQ; Microsoft SQL Server Provisioning; Backup and Restore support for ZooKeeper using KubeStash; OpsRequest and Autoscaler support for RabbitMQ; New version supports for MongoDB; Health Checker improvement for Memcached; and Grafana dashboard enhancement for Postgres. This post lists all the major changes done in this release since the last release. Find the detailed changelogs [HERE](https://github.com/kubedb/CHANGELOG/blob/master/releases/v2024.4.27/README.md). Let’s see the changes done in this release.
+
+## Druid
+
+This release introduces an enhanced monitoring feature for KubeDB-managed Druid deployments by integrating Grafana dashboards. These dashboards provide comprehensive insights into various Druid-specific metrics, statuses, as well as visual representations of memory and CPU consumption. With this dashboard, users can effortlessly assess the overall health and performance of their Druid clusters, enabling more informed decision-making and efficient management of resources.
+
+[Here](https://github.com/appscode/grafana-dashboards/tree/master/druid) is the step-by-step guideline
```

Comment:
Here is the step-by-step guideline -> Have to look here for the step-by-step guide to use the monitoring feature in Druid.